plugins/source/aws/resources/services/securityhub/findings.go

```diff
@@ -0,0 +1,30 @@
+package securityhub
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/securityhub/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func Findings() *schema.Table {
+	return &schema.Table{
+		Name:        "aws_securityhub_findings",
+		Description: `https://docs.aws.amazon.com/securityhub/1.0/APIReference/API_GetFindings.html`,
+		Resolver:    fetchFindings,
+		Transform:   transformers.TransformWithStruct(&types.AwsSecurityFinding{}, transformers.WithPrimaryKeys("AwsAccountId", "Region", "CreatedAt", "Description", "GeneratorId", "Id", "ProductArn", "SchemaVersion", "Title")),
+		Multiplex:   client.ServiceAccountRegionMultiplexer("securityhub"),
+		Columns: []schema.Column{
+			{
+				Name:     "request_account_id",
```

Comment:
Can `AwsAccountId` and `Region` have different values from the ones in the multiplexer?
Might be confusing to have both so maybe keep only the struct ones (even if different)?

Reply:
Yeah if I understood the PR description correctly, the values can be different from the one in the multiplexer, so I think this is a good compromise. @bbernays Maybe we can just add some details about this to the description of the table?

Reply:
Ah 🤦 I missed that. Maybe we can drop the `request_` ones for now?

Reply:
I think it depends: can security hub findings be different depending on the account/region the request is done from? If so, I think we should include the columns (and make them part of the PK?). If not, maybe we shouldn't be multiplexing on account and region

Reply:
Yes, Security Hub can aggregate findings from different regions based on user configuration

Reply:
I will add a comment in the table description. Are there any other changes needed?